### PR TITLE
Add: GitLab Support on Install

### PR DESCRIPTION
**Pull request by codemonkeyrawks**
Directions for using GitLab are similar to Travis CLI. This hopefully clears up on how to install it on GitLab.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ On Debian based distros:
 On Arch Linux based distros:
 
     pacman -S shellcheck
-
+
```
Comment (koalaman) [inline README.md]: There's some unnecessary trailing whitespace introduced here

File: README.md [hunk 2]
```diff
@@ -177,9 +177,11 @@ Distro packages already come with a `man` page. If you are building from source,
     pandoc -s -t man shellcheck.1.md -o shellcheck.1
     sudo mv shellcheck.1 /usr/share/man/man1
 
-## Travis CI
+## Travis CI & GitLab
+
+Travis CI has now intergrated ShellCheck by default, so you don't need to manually install it.
 
-Travis CI has now integrated ShellCheck by default, so you don't need to manually install it.
+GitLab does not intergrate ShellCheck by default, so you need to manually install it or pick what distro package to use.
```
Comment (koalaman) [inline README.md]: I'm not familiar with GitLab. Which distro is used? How do you pick the package?

**Comment by codemonkeyrawks** [comment]
Hello @koalaman:

A. I have updated the branch to remove the added whitespace that was added prior.
B. GitLab is not a Linux distribution: it is similar to Travis CLI where you use a .yaml file to define a build platform: it is based on docker.  I have attached an example below an temporary example I am working with.

Here is an example build script I have been messing around with:
https://github.com/codemonkeyrawks/Rose/blob/dev/.gitlab-ci.yml

Pipeline: just like a build in Travis CLI:
https://gitlab.com/codemonkeyrawks/Rose/-/jobs/165042622

I seem to be having some issues with this build in particular : some reason GitLab is returning an error instead of processing the command and moving on: I purposely created an error. So there is some differences between Travis CLI and GitLab I am working on weeding out.

**Comment by koalaman** [comment]
I see, thanks! I included this information as part of a larger rewrite in e45d81c8fa9

**Comment by frakman1** [comment]
@codemonkeyrawks The link you provided for a sample `.gitlab-ci.yml` no longer works. Can you post a new link or the code you used for us to see please?

**Comment by codemonkeyrawks** [comment]
@frakman1 please take a look at "https://gitlab.com/codemonkeyrawks/Debian/-/jobs/172277607" and "https://gitlab.com/codemonkeyrawks/Debian/-/blob/prod/.gitlab-ci.yml" for further code changes.

**Pull request closed without merging**